### PR TITLE
Basic Layout

### PR DESCRIPTION
**Pull request by sachreetkaur**
Added LeftSidebar Layout and dashboard view.

### CODE REVIEW

File: Pspcl/Pspcl.Web/Controllers/AccountController.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public async Task<IActionResult> Login(Pspcl.Models.LoginModel model)
                     await _userManager.UpdateAsync(user);
 
                     _logger.LogInformation($"User '{user.Email}' logged in.");
-                    return RedirectToAction("Index", "Home");
+                    return RedirectToAction("Index", "Dashboard");
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Controllers/AccountController.cs]: You can keep Dashboard action under HomeController, no need to create a separate new controller for a single Dashboard action.

File: Pspcl/Pspcl.Web/Controllers/DashboardController.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace Pspcl.Web.Controllers
+{
+    public class DashboardController : Controller
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Controllers/DashboardController.cs]: Not required

File: Pspcl/Pspcl.Web/Views/Dashboard/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+﻿@{
+    Layout = "~/Views/Shared/LeftSideBarLayout.cshtml";
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Views/Dashboard/Index.cshtml]: Wrong! It should be a partial view, HTML, HEAD, BODY tags should be under main layout view

File: Pspcl/Pspcl.Web/Views/Shared/_SidebarPartial.cshtml [hunk 4]
```diff
@@ -0,0 +1,61 @@
+﻿<div class="vertical-menu">
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Web/Views/Shared/_SidebarPartial.cshtml]: This should be under LeftSideBar

File: Pspcl/Pspcl.Web/Views/Home/Index.cshtml [hunk 5]
```diff
@@ -1,8 +1,10 @@
 ﻿@{
+    Layout = "~/Views/Shared/_TwoColumnLayout.cshtml";
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Web/Views/Home/Index.cshtml:2]: Name it proper --masterLayout

File: Pspcl/Pspcl.Web/Views/Shared/_LeftSidebar.cshtml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+                    <a href="@Url.Action("Login", "Account")">
+                        <i class="fab fa-stack-exchange"></i>
+                        <span>Issue Stock</span>
+                    </a>
+                </li>
+
+                <!-- Drop Down Sub Menu
+                    <li>
+                        <a href="javascript: void(0);" class="has-arrow">
+                           <i class="far fa-solid fa-building"></i>
```
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Web/Views/Shared/_LeftSidebar.cshtml:27]: Remove all commented code

File: Pspcl/Pspcl.Web/wwwroot/images/my-profile-pic1.jpg
Comment (VikasNischalDA) [inline Pspcl/Pspcl.Web/wwwroot/images/my-profile-pic1.jpg:1]: remove this pic instead use out line icon of person

**Review by VikasNischalDA — APPROVED** [review]

File: Pspcl/Pspcl.Web/Views/Home/Index.cshtml (hunk 5 above)
Comment (sandeepkumarlng) [reply Pspcl/Pspcl.Web/Views/Home/Index.cshtml:2]: Already discussed with Vikas. We can keep it as it is. @sachreetLng

File: Pspcl/Pspcl.Web/Views/Shared/_LeftSidebar.cshtml (hunk 6 above)
Comment (sandeepkumarlng) [reply Pspcl/Pspcl.Web/Views/Shared/_LeftSidebar.cshtml:27]: This is a partial view with all left sidebar content in it. @VikasNischalDA

**Review by sandeepkumarlng — APPROVED** [review]

**Review by sandeepkumarlng — APPROVED** [review]

**Pull request merged**